README.md

```diff
@@ -1,102 +1,38 @@
 # TypeSpec
 
-[Try TypeSpec Online](https://aka.ms/trytypespec)
+[Official Docs](https://typespec.io/) | [Try TypeSpec Online](https://aka.ms/trytypespec) | [Getting Started](https://typespec.io/docs) | [Language Overview](https://typespec.io/docs/language-basics/overview)
 
-TypeSpec is a language for describing cloud service APIs and generating other API
-description languages, client and service code, documentation, and other assets.
-TypeSpec provides highly extensible core language primitives that can describe API
-shapes common among REST, OpenAPI, GraphQL, gRPC, and other protocols.
+TypeSpec is a language for defining cloud service APIs and shapes. TypeSpec is a highly extensible language with primitives that can describe API shapes common among REST, OpenAPI, GraphQL, gRPC, and other protocols.
 
-Using TypeSpec, you can create reusable patterns for all aspects of an API, along with the ability to check for and flag known anti-patterns. These patterns establish "guardrails" for API designers and make it easier to follow best practices than deviate from them. TypeSpec promotes highly regular API designs that adhere to best practices by construction.
+TypeSpec is excellent for generating many different API description formats, client and service code, documentation, and many other assets. All this while keeping your TypeSpec definition as a single source of truth.
 
-You can try a work-in-progress build of the compiler by following the steps in
-the Getting Started section below. Please feel free to [file
-issues](https://github.com/microsoft/typespec/issues) for any issues you encounter while
-using the preview.
+Using TypeSpec, you can create reusable patterns for all aspects of an API using TypeSpec libraries. It has a rich linter framework with the ability to check for and flag known anti-patterns as well as an emitter framework that lets you be in control of the output to ensure it follows the patterns you want. Enabling the use of patterns establishes "guardrails" for API designers and makes it easier to follow best practices than to deviate from them. TypeSpec promotes highly quality and regular API designs that adhere to best practices by construction.
```

Comment:
```suggestion
Using TypeSpec, you can create reusable patterns for all aspects of an API and package those reusable patterns into libraries. These patterns establish "guardrails" for API designers and makes it easier to follow best practices than to deviate from them. TypeSpec also has a rich linter framework with the ability to flag anti-patterns as well as an emitter framework that lets you control of the output to ensure it follows the patterns you want. 
```

The proposed text felt out of order (talking about encapsulation, then emitter framework, then encapsulation again). I reordered and simplified, see what you think.